src/components/search.scss

```diff
@@ -0,0 +1,42 @@
+.search-component {
+  position: absolute;
+  width: 100%;
+  top: 0;
+}
+
+.search-arrow {
+  display: flex;
+  justify-content: center;
+  position: absolute;
+  padding-left: 23px;
+  top: 0;
+  bottom: 0;
+  flex-wrap: wrap;
+  align-content: space-around;
+}
+
+.search-input {
+  font-family: 'Apercu Pro';
```

Comment:
'Apercu Pro' is not a standard font family. If you use such families you need declare font face with source to the font